### DISCUSSION TITLE
Welcome to zim-desktop-wiki Discussions!

### DISCUSSION THREAD

**Discussion by jaap-karssenberg** [zim-desktop-wiki/zim-desktop-wiki#1569] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by quesada** [comment]
I'm Jose. I use zim as a 'second brain/zettelkasten' to think and make decisions. It's probably the most useful piece of software for me

**Comment by k4nzdroid** [comment]
Zim - The Persistent Zone of Your Brain.

**Comment by supertoothy** [comment]
I am George. I have enjoyed using Zim for many years. I like that it is so thoughtfully designed, and covers so many of my use cases. Over the years I have used it for maintaining my learning and notes, to manage a business, to organize my writing, to store important paperwork... this is very useful software.

While ZIm is simple to learn and more straightforward than many modern PKM softwares, I believe that new users may benefit from some walkthroughs of some use cases. I'd like to write some tutorials on my blog and share them here someday. By the way, it took some time to understand Zim. [This is the video](https://www.youtube.com/watch?v=yBZpWgzO9Ps) that made it click for me.
